openapi.yaml

```diff
@@ -255,6 +255,35 @@ paths:
           $ref: '#/components/responses/400'
         '500':
           $ref: '#/components/responses/500'
+  /servicenow/deductions:
+    post:
+      summary: Create payroll deductions
+      tags:
+        - deductions
+      description: Create payroll deductions
+      operationId: postDeductions
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/DeductionsPostBody'
+        description: Post body for payroll deductions
+        required: true
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DeductionsResult'
+        '400':
+          $ref: '#/components/responses/400'
+        '404':
```

Comment:
Under what circumstance, a 404 (resource not found) should be returned for this endpoint? You have to ensure your API design adheres to REST API principles. To be more specific, what is the "resource" of this endpoint? A person? a deduction? or something else?

Reply:
Well the resource is the deduction set. but the employee must exist in order for the deductions to be created. Would this still apply in that case?

Reply:
It depends, we might need to redesign this to make it work. I listed a few other questions [here](https://github.com/osu-mist/servicenow-api/pull/41#discussion_r1433196344) and they will help us to better understand how we should design this endpoint.

Reply:
Will discuss this one during our call